websockets/server.go

```diff
@@ -110,7 +110,7 @@ func (s *Server) Listen() {
 	go func() {
 		for {
 			s.Pulse()
-			time.Sleep(time.Second * 55)
+			time.Sleep(5 * time.Second)
```

Comment:
This one is every 5 seconds. The retry loop in the JS is every 10. Is this deliberate? If not, can it cause discrepancies?